README.md

```diff
@@ -1,96 +1,126 @@
+    
 # Loopchain
 
  Loopchain is a high-performance Blockchain Consensus & Network engine of ICON project.
-
- In order to run a loopchain node, you need to install [ICON Service](https://github.com/icon-project/icon-service) that runs a smart contract and interacts with loopchain engine, and [ICON RPC Server](https://github.com/icon-project/icon-rpc-server) that processes HTTP requests from clients. For details, refer to the guide below.
-
-## Installation
+ 
+ In order to run a loopchain node, you need to install [ICON Service] 
+that runs a smart contract and interacts with loopchain engine, 
+and [ICON RPC Server] that processes HTTP requests from clients. 
+ 
+ For details, refer to the guide below.
+
+## Table of Contents
+
+* [Getting Started](#getting-started)
+    + [Requirements](#requirements)
+    + [Installation](#installation)
+    + [TearDown](#teardown)
+* [See Also...](#see-also)
+    + [Documentation](#documentation)
+    + [License](#license)
+    
+## Getting Started
 
 ### Requirements
 
-Loopchain development and execution requires following environments.
-
-* OS: MacOS, Linux
-  * Windows are not supported yet.
+ Loopchain development and execution requires following environments.
 
-* Python
-
-  * Python 3.6.5+ or 3.7.x (recommended version)
-
-    **We will support 3.7.x only in future. Please upgrade python version to 3.7.x.**
-
-    Optional) We recommend to create an isolated Python 3 virtual environment with [virtualenv](https://virtualenv.pypa.io/en/stable/).
+* Python 3.6.5+ or 3.7.x (recommended version)
+  
+   We recommend to create an isolated Python 3 virtual environment with [virtualenv].
 
     ```bash
     $ virtualenv -p python3 venv
     $ source venv/bin/activate
     ```
 
-* Third party tools
-
-    ```
-    automake pkg-config libtool leveldb rabbitmq openssl
-    ```
-
-    If you're using package manager, you can install all of them through your package manager.
-
-    MacOS, for example)
+    > **_NOTE:_** We will support 3.7.x only in the future. Please upgrade python version to 3.7.x
     
+
+* Dependencies 
+ 
+  * MacOS
     ```bash
     $ brew install automake pkg-config libtool leveldb rabbitmq openssl
     $ brew services start rabbitmq
     ```
 
-* Check all requirements are installed and started properly
+  * Ubuntu
+    ```
+    $ sudo apt update
+    $ sudo apt install -y make build-essential libssl-dev zlib1g-dev libbz2-dev \
+      libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+      xz-utils tk-dev libffi-dev liblzma-dev automake libtool
+    ```
+    
+    **If you are using ubuntu 18.04, you need to install additional library below**
+    ```
+    $ sudo apt install -y libsecp256k1-dev
+    ```
+
+  * CentOS
+    ```
+    $ sudo yum update
+    $ sudo yum install git zlib-devel bzip2 bzip2-devel readline-devel sqlite sqlite-devel openssl-devel \
+      xz xz-devel libffi-devel gcc gcc-c++ automake libtool lsof -y
+    ```
```

Comment:
It would be better to add note not instruction to install `rabbitmq` depend on environment of user.

Reply:
Updated